bin/main

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+puts 'Tic-Tac-Toe : Terminal Edition'
+# Player1 Input
+puts 'Player 1, please enter your name'
+p1 = gets.chomp
+puts 'Player 2, please enter your name'
+p2 = gets.chomp
+puts "Welcome #{p1} and #{p2}"
+puts "#{p1}, X or O?"
+p1_xo = gets.chomp
+p2_xo = p1_xo == 'X' ? 'O' : 'X'
+puts "#{p1} is #{p1_xo}, #{p2} is #{p2_xo}"
+print '╔═══╦═══╦═══╗ \n ║ 1 ║ 2 ║ 3 ║ \n ╠═══╬═══╬═══╣ \n ║ 4 ║ 5 ║ 6 ║
+\n ╠═══╬═══╬═══╣ \n ║ 7 ║ 8 ║ 9 ║ \n ╚═══╩═══╩═══╝ \n'
+puts 'Instructions : Enter a number from 1 to 9 to use one of the boxes. \n'
+start = true
+while start == true
+  print '╔═══╦═══╦═══╗ \n ║   ║   ║   ║ \n ╠═══╬═══╬═══╣ \n ║   ║   ║   ║ \n ╠═══╬═══╬═══╣
```

Comment:
Since you are using single-quotes (`'`), the escape characters will not work. Use double-quotes (`"`).